chapters/publication.tex

```diff
@@ -99,91 +181,265 @@ \section{Collaborating on academic writing}
 
 \codeexample{pandoc.sh}{./code/pandoc.sh}
 
+\noindent The last portion after \texttt{csl=} specifies the bibliography style.
+You can download a CSL (Citation Styles Library) file\sidenote{
+  \url{https://github.com/citation-style-language/styles}}
+for nearly any journal and have it applied automatically in this process.
 Therefore, even in the case where you are requested to provide
 \texttt{.docx} versions of materials to others, or tracked-changes versions,
 you can create them effortlessly,
-and use Word's compare feature to generate a single integrated tracked version.
+and use external tools like Word's compare feature
+to generate integrated tracked versions when needed.
+
+\subsection{Getting started with \LaTeX\ in the cloud}
+
+\LaTeX\ is a challenging tool to get started using,
+but the control it offers over the writing process is invaluable.
+In order to make it as easy as possible for your team
+to use \LaTeX\ without all members having to invest in new skills,
+we suggest using a web-based implementation as your first foray into \LaTeX\ writing.
+Most such sites offer a subscription feature with useful extensions and various sharing permissions,
+and some offer free-to-use versions with basic tools that are sufficient
+for a broad variety of applications,
+up to and including writing a complete academic paper with coauthors.
+
+Cloud-based implementations of \LaTeX\ are suggested here for several reasons.
+Since they are completely hosted online,
+they avoids the inevitable troubleshooting of setting up a \LaTeX\ installation
+on various personal computers run by the different members of your team.
+They also typically maintain a single continuously synced master copy of the document
+so that different writers do not create conflicted or out-of-sync copies,
+or need to deal with Git themselves to maintain that sync.
+They typically allow inviting collaborators to edit in a fashion similar to Google Docs,
+though different services vary the number of collaborators and documents allowed at each tier.
+Most importantly, some tools provide a ``rich text'' editor
+that behaves pretty similarly to familiar tools like Word,
+so that collaborators can write text directly into the document without worrying too much
+about the underlying \LaTeX\ coding.
+Cloud services also usually offer a convenient selection of templates
+so it is easy to start up a project and see results right away
+without needing to know a lot of the code that controls document formatting.
+
+On the downside, there is a small amount of up-front learning required,
+continous access to the Internet is necessary,
+and updating figures and tables requires a bulk file upload that is tough to automate.
+One of the most common issues you will face using online editors will be special characters
+which, because of code functions, need to be handled differently than in Word.
+Most critically, the ampersand (\texttt{\&}), percent (\texttt{\%}), and underscore (\texttt{\_})
```

Comment:
Would this level of detail fit better into a wiki page than the main text of the book? it's a general question for us to keep in mind, i think the level of detail varies quite a bit across chapters.

Reply:
I think this might be important enough to include in the text, but maybe could be sidebarred

Reply:
moved to issue #348